_data/projects.yml

```
@@ -86,6 +86,13 @@
   contact: Michael Hirsch
   keywords: ["ionosphere_thermosphere_mesosphere","specific"]
 
+- name: HAPI
+  code: https://github.com/hapi-server/client-python
+  description: Python HAPI Client
+  logo: https://avatars0.githubusercontent.com/u/22482204?s=200&v=4
+  contact: Bob Weigel
+  keywords: ["data_retrieval","line_plots","plotting","spectra"]
```

Hi @rweigel, I think the only appropriate keyword for HAPI is data_retrieval. The HAPI package is not meant to be used to make line plots, or to provide plotting capability or analyze spectra, right?
```suggestion
  keywords: ["data_retrieval"]
```

---

Hi @ehsteve - There are two main functions in the `hapiclient` package - `hapi()`, which returns data, and `hapiplot()` which uses the data and metadata returned by `hapi()` to provide a proper plot of the data using the `hapi` metadata for labels, fill values, spectrum bin widths, etc. 

So `data_retrieval` is the primary keyword; I put the others in there because the package does provide non-trivial plotting functions. However, it is not a general-purpose plotting package. 

I'll let you decide what is appropriate given all of this.

---

That's fine. Sorry I forgot about that feature.